protected/humhub/modules/content/tests/codeception/unit/ReadableContentQueryTest.php

```diff
@@ -185,26 +187,84 @@ public function testPublicSpaceContentAsGuestNonGuestMode()
         $this->assertCount(0, $posts);
     }
 
-    public function testPublicSpaceContentAsAdminNotMemberCannotViewAllContent()
+    public function testPublicSpaceGlobalManageContentPermission()
     {
-        $this->becomeUser('AdminNotMember');
+        // User3 is member of Group ID 3
+        Group::findOne(3)->addUser(User::findOne(['username' => 'User3']));
```

Comment:
@yurabakhtin Any idea?

Reply:
@yurabakhtin Thank you very much for these options.
Moving the `Group::findOne(3)->addUser(User::findOne(['username' => 'User3']));` was the best one, because User2 don't have the same number of posts as User3 because of the ones created in the `_before()` method.

Done in commit https://github.com/humhub/humhub/pull/7329/commits/5365a55cf20d687829ad27c8b2863cd3787cfd48

Reply:
@marc-farre Ok, but if you wish you could add a new method like `self::addUserToGroup(2, 'User3')`.

Reply:
@yurabakhtin yes, good idea, added to commit https://github.com/humhub/humhub/pull/7329/commits/f77a4daf2faa512f04325968331580e84f5d487d

Reply:
@marc-farre Ok, thanks, but I thought to add the new method to the file `protected/humhub/tests/codeception/_support/HumHubDbTestCase.php` as we have for other methods like `setGroupPermission`, so the new method may be useful in future.

Reply:
@yurabakhtin I see, sorry, I'm still a newbie with self tests!
Done in commit https://github.com/humhub/humhub/pull/7329/commits/38ab50be19d9b0ce9d32bb712a976c82d2b98393